src/ovs/aca_vlan_manager.cpp

```diff
@@ -248,42 +248,26 @@ int ACA_Vlan_Manager::delete_l2_neighbor(string virtual_ip, string virtual_mac,
   return overall_rc;
 }
 
-// create a neighbor port without specifying vpc_id and neighbor ID
-int ACA_Vlan_Manager::create_neighbor_outport(alcor::schema::NetworkType network_type,
-                                              string remote_host_ip, uint /*tunnel_id*/,
-                                              ulong &culminative_time)
+string ACA_Vlan_Manager::get_zeta_gateway_id(uint tunnel_id)
```

Comment:
let's move Zeta specific code to zeta cpp file.

Reply:
@er1cthe0ne Is it possible to leave these zeta-specific codes in aca_vlan_manager as an interface for querying or modifying zeta_gateway_id in vpc_table_entry?

Reply:
we can keep it there for now, let's make sure they are focusing on get/set/remove/etc simple operations.